queries.yaml

```diff
@@ -125,7 +138,6 @@ layers:
     area-inclusion-threshold: 1
   pois:
     template: pois.jinja2
-    start_zoom: 4
```

Comment:
If that's the case, I think we should update the yaml to clamp these values. Should we do it in this pr? Or a subsequent one?